test/verify/check-kdump

```diff
@@ -421,8 +420,10 @@ class TestKdumpNFS(KdumpHelpers):
         self.machines["nfs"].write("/etc/exports", "/srv/kdump 10.111.113.0/24(rw,no_root_squash)\n")
         self.machines["nfs"].execute("mkdir -p /srv/kdump/var/crash; firewall-cmd --add-service nfs; systemctl restart nfs-server")
 
-        # ensure there is no local /var/crash, should not break kdump
-        m.execute("rmdir /var/crash")
+        # there shouldn't be any crash reports in the target directory
```

Comment:
This "clean up NFS test case" commit message is now obsolete: It doesn't "drop the disabling of kdump.service" any more.